hnn_core/network_builder.py

```diff
@@ -300,10 +294,14 @@ def _build(self):
 
         self._gid_assign()
 
+        # initialise vectors for synaptic currents based on simulation params
+        n_times = np.arange(0., self.net.params['tstop'],
+                            self.net.params['dt']).size + 1
```

Comment:
In support of this change, I can't think of any reason why `n_times` needs to be an attribute of `Network`. @blakecaldwell are there any GUI-related considerations we're missing here?

Reply:
I agree, it doesn't seem necessary. Thanks for notifying me @rythorpe.